### PR TITLE
Update licenses in opensource.html

### PR DESCRIPTION
**Pull request by Echolon**
Hello,

as the title says, update of licenses in `opensource.html`. 

Request by @FriedrichAltheide

### CODE REVIEW

File: Monal/opensource.html [hunk 1]
```diff
@@ -183,8 +309,1486 @@
...
+        * OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF
+        * SUCH DAMAGE.<br><br>
+        * 
+        * The licence and distribution terms for any publically available version or
+        * derivative of this code cannot be changed.  i.e. this code cannot simply be
+        * copied and put under another distribution licence
+        * [including the GNU Public Licence.]<br><br>
+        */
         
     </body>
```
Comment (FriedrichAltheide) [inline Monal/opensource.html:1794]: missing `</html>`

File: Monal/opensource.html [hunk 2]
```diff
@@ -86,10 +78,125 @@
...
+
+        3. Neither the name of Deusty nor the names of its contributors may be used to endorse or promote products derived from this software without specific prior written permission of Deusty, LLC.<br><br>
+
+        THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.<br><br>
+    <hr>
+    DEPENDENCY<br><br>
```
Comment (FriedrichAltheide) [inline Monal/opensource.html:119]: `<!—DEPENDENCY—>`

**Review by FriedrichAltheide — CHANGES_REQUESTED** [review]
- Update Monal Copyright to 2021
- Could you please check jrtplib again?

**Comment by Echolon** [comment]
> <hr> Success Checkmark By Icon Depot, IN from the Noun Project<br> https://thenounproject.com/search/?q=success&i=1276408 <br><br> <hr>

Whats this actually?

**Comment by FriedrichAltheide** [comment]
I think it is a image that we are using somewhere in the code
@tmolitor-stud-tu what do you think?

**Comment by Echolon** [comment]
I cannot find that it explicit states that it is MIT https://jrtplib.readthedocs.io/en/v3.11.1/index.html

**Comment by tmolitor-stud-tu** [comment]
I think so, too.

For the jrtplib i would check the source code directly, not readthedocs.

**Pull request merged**